packages/spark-core/components/_accordions.scss

```diff
@@ -26,6 +26,7 @@
 }
 
 .sprk-c-Accordion__summary {
+  align-items: center;
```

Comment:
Is this old code updates? Or fall?

Reply:
old, its a fix for aligning the old style accordions